### ISSUE TITLE
Add dietary restrictions tag

### ISSUE THREAD

**Issue by MatthewViens** [LukeSmithxyz/based.cooking#508]
As it is tags address the country of origin and main ingredient which is useful for identifying recipes based on flavor profile and ingredients on hand. There are also tags for fasting and cheesefare which I believe are a form of dietary restriction, but I think it would be more useful to add tags for more common dietary restrictions such as vegetarian, vegan, gluten-free, etc.

**Comment by Mentis-Moldvin** [comment]
> asting and cheesefare which I believe are a form of dietary restriction

it's not, its from religious restrictions

> but I think it would be more useful to add tags for more common dietary restrictions such as vegetarian, vegan, gluten-free, etc.
> more common

they are few and far between, obscure and stupid
the most common restrictions and prefferences are religious ones, there are more jews, christians and muslims than there are vegans, glutenbros or any other food-intolerant fad out there

just read the recipe and make your mind, if you want to include the pork in the pork chops or just eat the salad from the rest of it, its not that hard, dont look for pasta if you are gluten intolerant etc.

**Comment by DGKSK8LIFE** [comment]
I'm not really sure about religious-associated diets, though certain diets like "keto" or those for people with diabetes, for example, could be helpful as tags. I think Mentis, in his latter half response, makes a fair point, though there are so many recipes now that it's getting hard to navigate.

**Comment by MatthewViens** [comment]
To clarify, I didn't intend that the current dietary restrictions be replaced. My interpretation from the readme was that fasting and cheesefare were the only dietary tags that are recommended or approved. I imagine that I'm not alone in my interpretation as there are no other tags of the kind being used. Ultimately, what I'm suggesting is that if there is consensus that the usage of additional dietary restriction tags are beneficial, then the readme should be updated to encourage their use and addition.

**Comment by Mentis-Moldvin** [comment]
> To clarify, I didn't intend that the current dietary restrictions be replaced. My interpretation from the readme is that fasting and cheesefare are the only dietary tags that are recommended or approved. I imagine that I'm not alone in my interpretation as there are no other tags of the kind being used. Ultimately, what I'm suggesting is that if there is consensus that the usage of additional dietary restriction tags are beneficial, then the readme should be updated to encourage their use and addition.

to clarify my stand on the issue:
1. ban veganism
2. ban vegans and purposely vegan diets
3. ban vegetarians and purposely vegetarian diets
4. if any religious restriction coexists within the frame of vegan/vegetarian its only coincidence and its ok
 
I dont support the idea of promoting veganism or the vegan lifestyle by any means, not only that I am against it. Zero tolerance when it comes to those ideas and their logical consequences.

Read more here: https://lukesmith.xyz/articles/vegan

**Comment by siggines** [comment]
No.

**Comment by siggines** [comment]
>vegetarian, vegan, gluten-free, etc.
 No.

**Comment by GhostVIRUS** [comment]
Imagine being so politically-polarized that you segregate all the people who have different ethical or physical needs from contributing. What's the point in banning tags rather than pure hatred towards other people? The same goes for that "tradition" thing. How do you actually define it? How old should a "traditional" recipe be? Mammoth ribs fried at open fire only, or will a grandma's recipe suffice? How traditional is chicken (India) or pork (Eurasia) in America?

This intolerance is sickening. I live in a poor 3rd world country and it fills me with despair to see all this on the topic of freaking COOKING in the "richest country in history". What would you people be willing to do to others when it comes to more substantial disagreements? That is terrifying.

**Comment by Mentis-Moldvin** [comment]
> Imagine being so politically-polarized that you segregate all the people who have different ethical or physical needs from contributing. What's the point in banning tags rather than pure hatred towards other people? The same goes for that "tradition" thing. How do you actually define it? How old should a "traditional" recipe be? Mammoth ribs fried at open fire only, or will a grandma's recipe suffice? How traditional is chicken (India) or pork (Eurasia) in America?
> 
> This intolerance is sickening. I live in a poor 3rd world country and it fills me with despair to see all this on the topic of freaking COOKING in the "richest country in history". What would you people be willing to do to others when it comes to more substantial disagreements? That is terrifying.

I live in a 3rd world country too, brother. I hate the white privileged westerners who eat bugs for the joy of it and call it veganism or whatever, if you have medical condition or are too poor to afford food - the choices are obvious - you listen to your doctor and the diet he gives you, or you cut off the interwebs and get some proper food instead of look at gluten free beef for vegans.
Simple as.
There is nothing traditional to hipster culture - its progressive in nature, the tofu eaters have no traditions and culture. Chicken and Pork are pretty traditional, but trying to find a vegan recipe for those two - thats something new.

**Comment by Mentis-Moldvin** [comment]
> > vegetarian, vegan, gluten-free, etc.
> 
> No.
> I suppose a meat lacking tag may be useful, with the exemption of obvious instances such as smoothies or what-the-shit. A "meat-less" tag may-be ...
> Or "rice-based", "veg-based", and "bread-based"?
> It's hard to think of many more bases for a vegan diet than those, so they would be just as useful to a vegan? I suppose I left out Soy.

yes exactly, just search for vegan-like recipes and maybe add meat-less for meals who are not obviously without meat
but the idea of adding vegan and non-vegan etc. thats a slippery slope
if you wish to eat bread, but you want it gluten free - scroll through the 2 or 3 recipes and find out if the bread is gluten free, it might be in the name even, the chances are low - same goes for the pork chops, what are the chances of the pork chops being vegan? why did you search for vegan pork chops anyway? maybe vegan pasta is something you'd want? well just scroll trough the 10ish recipes on pasta and see if its there
idk I just find it stupid and unnecessary, with the risk of promoting bad diet - vegan diet

**Comment by Dko1905** [comment]
Let's just close this issue, there are too many different strong opinions, I don't think it will end well if we continue.
